release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3508,3 +3560,24 @@ link:https://access.redhat.com/solutions/6833311[{product-title} 4.6.56 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-6-57"]
+=== RHBA-2022:1621 - {product-title} 4.6.57 bug fix and security update
+
+Issued: 2022-05-04
+
+{product-title} release 4.6.57, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1621[RHBA-2022:1621] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:1620[RHSA-2022:1620] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6956136[{product-title} 4.6.57 container image list]
+
+[id="ocp-4-6-57-removed-feature-azure-mint-mode"]
+==== Removed features
+
```

Comment:
Should we follow the same formatting conventions as seen here, with a subheading for the feature being remove? https://deploy-preview-45282--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-removed-features

This is the first case I've seen of a feature removal. If you've done this in other branches, this is fine.

Reply:
Yes, we did it in other branches. Here is when we added it for [4.9 ](https://github.com/openshift/openshift-docs/pull/43289/files#diff-29676acc41893298b1c8ef894ab212568eb82a2247e870cac8cb2f42624eaac3R2654)for example